### PR TITLE
Milestone 5 : Write test for controllers

### PR DESCRIPTION
**Pull request by Chrissiku**
## Project requirements 🟢 
- [x] Add RSpec as a gem to your project.
- [x] Create a Request spec file for all your controllers.
- [x] Make sure that for each action you check:
    - If the response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ gem 'bootsnap', require: false
 
 group :development, :test do
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
```
Comment (mateo951) [inline Gemfile:34]: Has and uses RSpec gem for testing ✔

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -2,6 +2,36 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'Return http success' do
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:5]: Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 🐐Well done!

File: spec/requests/posts_spec.rb [hunk 3]
```diff
@@ -2,6 +2,36 @@
...
+
+    it 'Should render the index template' do
+      get '/posts/show'
+      expect(response).to render_template(:show)
+    end
+
+    it 'Should include the correct placeholder' do
+      get '/posts/show'
+      expect(response.body).to include('Single user posts')
+    end
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:35]: `index` and `show` actions for `User` and `Post` controllers are tested ✔

File: spec/requests/posts_spec.rb [hunk 4]
```diff
@@ -2,6 +2,36 @@
...
+
+  describe 'GET /show' do
+    it 'Return http success' do
+      get '/posts/show'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'Should render the index template' do
+      get '/posts/show'
+      expect(response).to render_template(:show)
```
Comment (mateo951) [inline spec/requests/posts_spec.rb:29]: Spec file include tests that ensures correct view template is rendered 🙌

**Review by mateo951 — APPROVED** [review]
### Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chrissiku** [comment]
Thank you @mateo951

**Pull request merged**